build.gradle.kts

```diff
@@ -232,6 +232,7 @@ dependencies {
     testImplementation("com.zaxxer:HikariCP:4.0.3")
 
     implementation("com.amazonaws:aws-java-sdk-rds:1.12.128")
+    implementation("com.amazonaws:aws-java-sdk-secretsmanager:1.12.213")
```

Comment:
This dependency should be runtime-only dependency. 
https://dzone.com/articles/optional-dependencies